content/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "com.adobe.cq.core.wcm.components",
+    "description": "Adobe Experience Manager Core WCM Components Content Package",
+    "version": "1.0.0",
```

Comment:
Shouldn't we align the version of the npm module with the maven version?

Reply:
@bpauli - we should, but we have to sync with the pom version somehow

Reply:
@bpauli you could use https://www.npmjs.com/package/sync-pom-version-to-package